deps/rabbitmq_stomp/src/rabbit_stomp.erl

```diff
@@ -28,6 +28,7 @@
 start(normal, []) ->
     Config = parse_configuration(),
     Listeners = parse_listener_configuration(),
+    rabbit_global_counters:init([{protocol, stomp}]),
```

Comment:
I'd vote to differentiate by STOMP versions, i.e. 1.0, 1.1, 1.2 since that's what we currently do for MQTT (3.1, 3.1.1, 5.0).

Reply:
Where are the global counters per protocol and queue type initialised? Are they missing?